x-pack/plugins/cases/public/containers/api.ts

```diff
@@ -182,9 +183,18 @@ export const getCases = async ({
   const query = {
     ...(filterOptions.status !== StatusAll ? { status: filterOptions.status } : {}),
     ...(filterOptions.severity !== SeverityAll ? { severity: filterOptions.severity } : {}),
-    assignees: filterOptions.assignees,
-    reporters: constructReportersFilter(filterOptions.reporters),
-    tags: filterOptions.tags,
+    ...(filterOptions.assignees === null || filterOptions.assignees.length > 0
```

Comment:
Could we move this to its own function. It's a bit hard to read inlined.

something like
```
...constructAssigneesFilter()
```